src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
+    var bbs = csPrimitives.context.billboards;
+    var opacity = olLayer.getOpacity();
+    if (!goog.isDef(opacity)) {
+      opacity = 1;
+    }
+    bbs.olLayerOpacity = opacity;
+    var i, bb;
+    for (i = bbs.length - 1; i >= 0; --i) {
+      bb = bbs.get(i);
+      bb.color = new Cesium.Color(1.0, 1.0, 1.0, bb.olStyleOpacity * opacity);
```

Comment:
However, `Cesium.Color.fromAlpha` is not available yet in the Cesium version we currently use.